packages/suite/src/components/suite/modals/AddAccount/components/AddAccountButton.tsx

```diff
@@ -1,70 +1,84 @@
-import React from 'react';
-import { Button, Tooltip } from '@trezor/components';
-import { Account, Network } from '@wallet-types';
+import React, { useCallback } from 'react';
+import { Button, TooltipConditional } from '@trezor/components';
+import { Account } from '@wallet-types';
 import { Translation } from '@suite-components';
 import { useAnalytics, useAccountSearch } from '@suite-hooks';
 
 interface Props {
-    network: Network;
-    accounts: Account[];
+    emptyAccounts: Account[];
     onEnableAccount: (account: Account) => void;
 }
```

Comment:
nit:
move interface directly above its usage (`const AddAccountButton = ...`)

Reply:
I like the main interface on top, but no strong opinion here, it definitely makes sense to have it close to the usage. Will move 👌

Reply:
https://github.com/trezor/trezor-suite/pull/3897/commits/f0c2bcb06883bacf7d1080c829470fac66956d06